boards.txt

```diff
@@ -1991,3 +1991,25 @@ Midatronics.menu.rtlib.nanofps=Newlib Nano + Float Printf/Scanf
 Midatronics.menu.rtlib.nanofps.build.flags.ldspecs=--specs=nano.specs -u _printf_float -u _scanf_float
 Midatronics.menu.rtlib.full=Newlib Standard
 Midatronics.menu.rtlib.full.build.flags.ldspecs=
+
+###############################
+# ThunderPack
+ThunderPack.name=ThunderPack
+ThunderPack.build.core=arduino
+ThunderPack.build.product_line=STM32L072xx
+ThunderPack.build.board=ThunderPack
+ThunderPack.build.variant=THUNDER_PACK
+ThunderPack.build.mcu=cortex-m0plus
+ThunderPack.build.series=STM32L0xx
+ThunderPack.build.cmsis_lib_gcc=arm_cortexM0l_math
+ThunderPack.build.vid=0x0483
+ThunderPack.build.extra_flags={build.usb_flags} -DUSBD_USE_CDC -D{build.product_line} {build.enable_usb} {build.xSerial} -D__CORTEX_SC=0
+
+ThunderPack.upload.maximum_size=196608
+ThunderPack.upload.maximum_data_size=20480
+
+ThunderPack.menu.upload_method.dfuUtil=DFU Util
```

Comment:
Please do a separate PR for this.
Note that `linux64` directory has been removed as Arduino IDE does not manage it at runtime.
So you will have to provide a script to properly call the right binary depending of the arch.

Reply:
✔️

Reply:
That's good to know. Is there a good model of this kind of script I should follow? I assume this would go in the `Arduino_Tools` repository?

Reply:
Yes, it has been updated on the master and available on next release

Reply:
Great. Per your recommendation: https://github.com/stm32duino/Arduino_Tools/pull/46